### ISSUE TITLE
Allow users to access MIDI 14bit msb / lsb CC offset variable

### ISSUE THREAD

**Issue by ayalavalva** [intechstudio/grid-editor#994]
It would be extremely convenient for users to have access to a variable to control the MIDI 14bit msb / lsb CC offset. 

A fixed CC offset of 32 is is extremely cumbersome to deal with, especially when using 4 controllers featuring MIDI14bit elements on a single MIDI channel.

This CC offset variable would provide a lot more flexibility to Intech users to span their controllers across the full 127 CC range of a single MIDI channel.

**Comment by SukuWc** [comment]
This offset comes from the midi specifiaction as far as I know

**Comment by ayalavalva** [comment]
Yes, but it is still useful for programming environments such as Pure Data, SuperCollider, Chuck, etc.

Actually the offset can be controlled this way for each elements / group of elements:

midi_send(ch, 176, cc, val // 128)
midi_send(ch, 176, cc + 16, val % 128)

It's not possible to see the MIDI 14bit data in the MIDI monitor though.

**Comment by SukuWc** [comment]
Cannot implement it now, use codeblock with two separate midi_send() calls
